src/main/java/org/folio/gobi/LookupService.java

```diff
@@ -68,19 +69,26 @@ public LookupService(RestClient restClient) {
    * @param location
    * @return UUID of the location
    */
-  public CompletableFuture<String> lookupLocationId(String location) {
+  public CompletableFuture<LocationTranslationResult> lookupLocationId(String location) {
     logger.debug("lookupLocationId:: Trying to look up locationId by location '{}'", location);
-    String query = HelperUtils.encodeValue(String.format(CQL_CODE_STRING_FMT, location));
-    String endpoint = String.format(LOCATIONS_ENDPOINT + QUERY, query);
-    return restClient.handleGetRequest(endpoint).toCompletionStage().toCompletableFuture()
+    return restClient.handleGetRequest(LOCATIONS_ENDPOINT + "?limit=1000").toCompletionStage().toCompletableFuture()
```

Comment:
limit can be deleted, default limit is 10000 will be used when this story will be completed: https://folio-org.atlassian.net/browse/MSEARCH-812

Reply:
removed